ci/axis/tests.yaml

```
@@ -21,6 +21,6 @@ LINUX_VER:
   - centos8
 
 PYTHON_VER:
-  - 3.7
+  - 3.9
```

```suggestion
  - 3.8
```

Let's switch this to `3.8` to continue testing on the oldest version of Python that we support.

---

Change completed